src/raygun.js

```diff
@@ -445,6 +445,16 @@ var raygunFactory = function (window, $, undefined) {
         return true;
     }
 
+    function contains(array, obj) {
```

Comment:
It's sad that Array.prototype.includes isn't widely supported : ( This looks solid though.